### PR TITLE
doc: smr-disk.md - "Checking the Disk Info'

### PR DESCRIPTION
**Pull request by zdover23**
This PR improves the flow of the English in
"Checking the Disk Info" in the "Serial ATA
ZAC Disks and SATA Host Controllers" section
of the "Getting Started with SMR Disks" chapter
of the Getting Started Guide.

Signed-off-by: Zac Dover <zac.dover@gmail.com>

### CODE REVIEW

File: docs/getting-started/smr-disk.md [hunk 1]
```diff
@@ -115,24 +115,24 @@ sd 5:0:0:0: [sdb] Attached SCSI disk
...
-host managed. The total number of zones of the disk is also displayed. In this
-example, the disk capacity is 15 TB and has 55880 zones.
+The zone model of the disk is confirmed to be "host managed". The total number
+of zones on the disk is also displayed. In this example, the disk capacity is
+15 TB and the disk has 55880 zones.
 
-The zone size of the disk can also be inspected through sysfs, with the
-attribute *chunk_sectors*.
+The zone size of the disk can be inspected by using sysfs to examine the
+attribute *chunk_sectors*:
```
Comment (damien-lemoal) [inline docs/getting-started/smr-disk.md:123]: This is very convoluted... Maybe something simpler like this ?

The zone size of the disk can be discovered using the *chunk_sectors* sysfs attribute file:

Comment (zdover23) [reply docs/getting-started/smr-disk.md]: I see your point.

I've tried to make it yet less convoluted.

Let me know what you think.

File: docs/getting-started/smr-disk.md [hunk 2]
```diff
@@ -115,24 +115,25 @@ sd 5:0:0:0: [sdb] Attached SCSI disk
...
-host managed. The total number of zones of the disk is also displayed. In this
-example, the disk capacity is 15 TB and has 55880 zones.
+The kernel ring buffer message " sd 5:0:0:0: [sdb] Host-managed zoned block device" confirms that the zone model of the disk is "host managed". 
 
-The zone size of the disk can also be inspected through sysfs, with the
-attribute *chunk_sectors*.
+The total number of zones on the disk is also displayed. In this example, the
+disk capacity is 15 TB and the disk has 55880 zones.
+
+Check the zone size of the disk by reading the value of
```
Comment (zdover23) [inline docs/getting-started/smr-disk.md:123]: Here's the updated text that was previously more convoluted.

Comment (zdover23) [reply docs/getting-started/smr-disk.md:123]: @damien-lemoal ^^^

File: docs/getting-started/smr-disk.md [hunk 3]
```diff
@@ -89,7 +89,7 @@ The possible values of the *zoned* attribute are shown in the table below.
 
 </center>
 
-The kernel messages also include useful information on the disk.
+Kernel-ring-buffer messages also contain useful information about the disk:
```
Comment (damien-lemoal) [inline docs/getting-started/smr-disk.md:92]: Please remove "ring-buffer". This is a kernel implementation detail that is not relevant here.

File: docs/getting-started/smr-disk.md [hunk 4]
```diff
@@ -115,24 +115,25 @@ sd 5:0:0:0: [sdb] Attached SCSI disk
 ...
 ```
 
-Among other information, the zone model of the disk is confirmed as
-host managed. The total number of zones of the disk is also displayed. In this
-example, the disk capacity is 15 TB and has 55880 zones.
+The kernel ring buffer message " sd 5:0:0:0: [sdb] Host-managed zoned block device" confirms that the zone model of the disk is "host managed".
```
Comment (damien-lemoal) [inline docs/getting-started/smr-disk.md:118]: Same here. We do not need "ring buffer".

File: docs/getting-started/smr-disk.md (hunk 3 above)
Comment (zdover23) [reply docs/getting-started/smr-disk.md:92]: I've removed it.

File: docs/getting-started/smr-disk.md (hunk 4 above)
Comment (zdover23) [reply docs/getting-started/smr-disk.md:118]: I've removed it.

**Comment by damien-lemoal** [comment]
The patch looks broken. It seems that you have multiple patches squashed together into a single one, so multiple titles and signed-off-by. Please fix this to have a single PR with multiple patches.

**Comment by zdover23** [comment]
Roger that, Damien.

I'll fix it later today.

**Pull request closed without merging**